ui/src/app/cohort-review/page-layout/page-layout.ts

```diff
@@ -34,7 +51,7 @@ export class PageLayout implements OnInit, OnDestroy {
     }).then(review => {
       cohortReviewStore.next(review);
       this.reviewPresent = review.reviewStatus !== ReviewStatus.NONE;
-      if (this.reviewPresent) {
+      if (this.reviewPresent && this.router.url.split('/').pop() === 'review') {
```

Comment:
I would ask same question as above. Can we use state to determine this or do we have to check url strings?

Reply:
The reason I added this condition was to prevent switching back to the `participants` route when you refresh the page on the `participants/:pid` route (participant detail page), in which case the state gets reset anyway.

It's definitely not the cleanest solution, but the only way you can get the current route in Angular is using the url.